### DISCUSSION TITLE
Have an option to use the old sprites for highscores

### DISCUSSION THREAD

**Discussion by Stickaplex** [runelite/runelite#18637] [Ideas]
While the new ones look good, i just find them far less intuitive as the old ones used in-game ones so it was familiar. With the new ones i find I have to hover over them to know what they are.

**Comment by Felanbird** [comment]
Implementing the old hiscore icons was done manually and generally the process was the sort of thing devs dislike having to do. Using the official™️ ones Jagex supplies is much more hands-off, so we're not really interested in supporting the old ones anymore.

**Reply by Stickaplex** [reply]
I can respect that, but are these really the official ones? Looking at the official highscores they dont seem to match up to any icon. Don't think I've seen any of these anywhere

**Reply by Felanbird** [reply]
They are the official™️ mobile sprites, and because of this, we can pull them directly from the client.

**Reply by Stickaplex** [reply]
ahhh ok mobile. explains why ive never seen them before

**Comment by tomtargosz** [comment]
Would love to see a toggle for this. Understandable to have it default to the official sprites, but truthfully they look awful.

**Reply by Felanbird** [reply]
There's nothing to toggle the old sprites no longer exist in RuneLite.
